benchmark/framework/containerd_utils.go

```diff
@@ -138,11 +137,8 @@ func (proc *ContainerdProcess) DeleteImage(ctx context.Context, imageRef string)
 
 func (proc *ContainerdProcess) CreateContainer(
 	ctx context.Context,
-	image containerd.Image,
-	opts ...containerd.NewContainerOpts) (containerd.Container, func(), error) {
+	opts []containerd.NewContainerOpts) (containerd.Container, func(), error) {
```

Comment:
Any particular reason for changing the header this way?

Reply:
Whoops. At one point I was playing around with this function taking both `[]containerd.NewContainerOpts` and `[]oci.SpecOpts` so they were both slices instead of varargs. That ended up being confusing and error prone, so I went back to this signature, but forgot to switch the type back. Fixed in the next commit.